bitnami/apache/values.yaml

```diff
@@ -1,8 +1,11 @@
-## Global Docker image registry
-## Please, note that this will override the image registry for all the images, including dependencies, configured to use the global value
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry and imagepullSecrets
 ##
 # global:
 #   imageRegistry:
```

Comment:
should we not add myRegistryName here as well?

Reply:
Sure. We can add it.